views/api.go

```diff
@@ -29,35 +29,47 @@ type (
 )
 
 // SetTokenHandler sets a valid JWT in a cookie instead of returning a string
-func (v *Views) SetTokenHandler(w http.ResponseWriter, r *http.Request) {
-	//fmt.Println("DEBUG - SET TOKEN")
-	session, _ := v.cookie.Get(r, v.conf.SessionCookieName)
-	u := helpers.GetUser(session)
+func (v *Views) SetTokenHandler(c echo.Context) error {
+	session, _ := v.cookie.Get(c.Request(), v.conf.SessionCookieName)
+	c1 := v.getSessionData(session)
```

Comment:
NITPICK: could `getSessionData` take an `echo.Context` and handle the cookie business internally? I know it only saves one line, but it'll be used all over the place.